### PR TITLE
Updates the example CorDapp page.

### PR DESCRIPTION
**Pull request by joeldudleyr3**

### CODE REVIEW

File: docs/source/deploying-a-node.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      ...
+
+      // Cordapp dependencies
+      // Specify your cordapp's dependencies below, including dependent cordapps
+  }
+
+For further information about managing dependencies, see
+`the Gradle docs <https://docs.gradle.org/current/userguide/dependency_management.html>`_.
+
+Cordform and deployNodes
```
Comment (GavT) [inline docs/source/deploying-a-node.rst]: Can this title be more friendly? e.g. Using deployNode to build a local Corda network

File: docs/source/deploying-a-node.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
+Deploying a node
+================
+
+Building nodes using Gradle
+---------------------------
+Nodes are usually built using a Gradle task. The cannonical Gradle file for building nodes is the one used by the
```
Comment (GavT) [inline docs/source/deploying-a-node.rst]: Typo: canonical

File: docs/source/deploying-a-node.rst [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            rpcPort 10012
+            webPort 10013
+            cordapps = []
+            rpcUsers = [[ user: "user1", "password": "test", "permissions": []]]
+        }
+    }
+
+You can extend ``deployNodes`` to generate any number of nodes you like. The only requirement is that you must specify
+one node as running the network map service.
+
```
Comment (GavT) [inline docs/source/deploying-a-node.rst]: Can you reference how one node was specified as running the network map service in the example you give above. I guess its the Controller node

File: docs/source/deploying-a-node.rst [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+.. sourcecode:: none
+
+    . nodeName
+    ├── corda.jar       // The Corda runtime
+    ├── node.conf       // The node's configuration
+    └── plugins         // Any installed CorDapps
+
+.. note:: Outside of development environments, do not store your node directories in the build folder.
+
+If you make any changes to your ``deployNodes`` task, you will need to re-run the task to see the changes take effect.
```
Comment (GavT) [inline docs/source/deploying-a-node.rst]: What about deploying a node by simply grabbing the latest corda and corda-webserver jars from maven and saving them to a dir on your local machine? This is what I do!

File: docs/source/getting-set-up.rst [hunk 5]
```diff
@@ -158,11 +158,14 @@ instead by running ``git checkout release-M12``.
 
 Next steps
 ----------
+The best way to check that everything is working fine is by running the :doc:`tutorial CorDapp <tutorial-cordapp>` and
+the :doc:`samples <running-the-demos>`.
```
Comment (GavT) [inline docs/source/getting-set-up.rst]: All these doc links look incorrect when rendering the final page in a browser. Can you check the nomenclature

File: docs/source/running-a-node.rst [hunk 6]
```diff
@@ -1,38 +1,20 @@
...
-
-.. warning:: Also note that the bootstrapping process of the ``corda.jar`` unpacks the Corda dependencies into a
-   temporary folder. It is therefore suggested that the CAPSULE_CACHE_DIR environment variable be set before
-   starting the process to control this location.
-
 Starting your node
 ------------------
 
-Now you have a node server with your app installed, you can run it by navigating to ``<node_dir>`` and running:
+After following the steps in :doc:`deploying-a-node`, you should have deployed your node(s) with any chosen CorDapps
```
Comment (GavT) [inline docs/source/running-a-node.rst]: Again, can you check this doc link is formatted correctly. Looks FUBAR'ed when viewing in browser

File: docs/source/tutorial-cordapp.rst [hunk 7]
```diff
@@ -7,832 +7,569 @@
...
-By default, the ``master`` branch of the ``cordapp-tutorial`` points to a SNAPSHOT release of Corda, this is because it is
-being constantly updated to reflect the changes in the master branch of the `corda` repository.
-
-.. note:: If you wish to use a SNAPSHOT release then follow the instructions below: `Using a SNAPSHOT release`_.
-
-To clone the ``cordapp-tutorial`` repository, use the following command:
+Downloading the example CorDapp
+-------------------------------
+If you haven't already, set up your machine by following the :doc:`quickstart guide <getting-set-up>`. Then clone the
+example CorDapp from the `cordapp-tutorial repository <https://github.com/corda/cordapp-tutorial>`_ using the following
```
Comment (GavT) [inline docs/source/tutorial-cordapp.rst]: check the :doc: nomenclature

File: docs/source/tutorial-cordapp.rst [hunk 8]
```diff
@@ -7,832 +7,569 @@
...
-dialogue, simply close and re-open IntelliJ again to see it again.
-
-**Alternative approach**
+Click the 'import gradle project' link. A dialogue will pop-up. Press OK. Gradle will now download all the
+project dependencies and perform some indexing. This usually takes a minute or so.
 
-Alternatively, one can instruct IntelliJ to create a new project through cloning a repository. From the IntelliJ welcome
-dialogue (shown above), opt to 'check out from version control', then select git and enter the git URL for the example CorDapp 
-(https://github.com/corda/cordapp-tutorial). You'll then need to import the Gradle project when prompted, as explained above.
+If you the 'import gradle project' pop-up does not appear, click the small green speech bubble at the bottom-right of
```
Comment (GavT) [inline docs/source/tutorial-cordapp.rst]: Typo: "If you the"

File: docs/source/deploying-a-node.rst (hunk 1 above)
Comment (joeldudleyr3) [inline docs/source/deploying-a-node.rst]: This isn't the part where we actually deploy the nodes, but I'll change the name a bit to be less threatening.

File: docs/source/deploying-a-node.rst (hunk 2 above)
Comment (joeldudleyr3) [inline docs/source/deploying-a-node.rst]: Fixed

File: docs/source/deploying-a-node.rst (hunk 3 above)
Comment (joeldudleyr3) [inline docs/source/deploying-a-node.rst]: Thanks, done

File: docs/source/tutorial-cordapp.rst (hunk 8 above)
Comment (joeldudleyr3) [inline docs/source/tutorial-cordapp.rst]: Thanks, fixed

File: docs/source/tutorial-cordapp.rst [hunk 9]
```diff
@@ -7,832 +7,569 @@
...
+    second: "(observable)"
+
+The same states and transactions will be present on NodeB, who was NodeA's counterparty in the creation of the IOU.
+However, the vault and local storage of NodeC will remain empty, since NodeC was not involved in the transaction.
+
+Via the h2 web console
+~~~~~~~~~~~~~~~~~~~~~~
```
Comment (GavT) [inline docs/source/tutorial-cordapp.rst]: should viewing the h2 db content be broken out as a separate rst? It is useful beyond just the tutorial. This tutorial could reference the h2 page

File: docs/source/tutorial-cordapp.rst [hunk 10]
```diff
@@ -7,832 +7,569 @@
...
-only requirement is that you must specify a node to run as the network map service and one as the notary service.
+  ``Database connection URL is              : jdbc:h2:tcp://10.18.0.150:56736/node``
 
-.. note:: CorDapps in the current cordapp-tutorial project are automatically registered with all nodes defined in
-  ``deployNodes``, although we expect this to change in the near future.
+* Paste this string into the JDBC URL field and click Connect.
 
-.. warning:: Make sure that there are no port clashes!
+You will be presented with a web interface that shows the contents of your node's storage and vault, and provides an
+interface for you to query them using SQL.
```
Comment (GavT) [inline docs/source/tutorial-cordapp.rst]: No credentials to provide?

File: docs/source/getting-set-up.rst (hunk 5 above)
Comment (joeldudleyr3) [inline docs/source/getting-set-up.rst]: They work, just GitHub doesn't render them properly.

File: docs/source/running-a-node.rst (hunk 6 above)
Comment (joeldudleyr3) [inline docs/source/running-a-node.rst]: They work, just GitHub doesn't render them properly.

File: docs/source/tutorial-cordapp.rst (hunk 7 above)
Comment (joeldudleyr3) [inline docs/source/tutorial-cordapp.rst]: They work, just GitHub doesn't render them properly.

File: docs/source/tutorial-cordapp.rst [hunk 11]
```diff
@@ -7,832 +7,569 @@
...
 
-Once you have made some changes to your CorDapp you can redeploy it with the following command:
+You can close the application using ``ctrl+C``.
 
-Unix/Mac OSX: ``./gradlew deployNodes``
+For more information on the client RPC interface and how to build an RPC client application, see:
 
-Windows: ``gradlew.bat deployNodes``
+* :doc:`Client RPC documentation <clientrpc>`
+* :doc:`Client RPC tutorial <tutorial-clientrpc-api>`
```
Comment (GavT) [inline docs/source/tutorial-cordapp.rst]: check :doc: renders ok

File: docs/source/tutorial-cordapp.rst (hunk 10 above)
Comment (joeldudleyr3) [inline docs/source/tutorial-cordapp.rst]: You use the defaults - I've made that clearer.

File: docs/source/tutorial-cordapp.rst (hunk 11 above)
Comment (joeldudleyr3) [inline docs/source/tutorial-cordapp.rst]: It does.

**Pull request merged**